packages/amplify-e2e-core/src/categories/api.ts

```diff
@@ -719,7 +719,7 @@ export function rebuildApi(projDir: string, apiName: string) {
 export function addRestContainerApiForCustomPolicies(projectDir: string, settings: { name: string }) {
   return new Promise<void>((resolve, reject) => {
     spawn(getCLIPath(), ['add', 'api'], { cwd: projectDir, stripColors: true })
-      .wait('Please select from one of the below mentioned services:')
+      .wait('Select from one of the below mentioned services:')
```

Comment:
I think we reverted this change? Can you confirm once

Reply:
I see it in extOverrides

Reply:
![image](https://user-images.githubusercontent.com/56042290/140626952-bbd312b8-1212-4f04-b433-4ca2f65501dd.png)